build-ps/percona-server.spec

```diff
@@ -542,7 +542,7 @@ mkdir debug
            -DWITH_LZ4=bundled \
            -DWITH_ZLIB=bundled \
            -DWITH_ZSTD=bundled \
-           -DWITH_READLINE=system \
+           -DWITH_EDITLINE=bundled \
```

Comment:
Don't we need to explicitly specify `-DWITH_EDITLINE=bundled` for debian package specs as well?

Reply:
I kept `-DWITH_EDITLINE=bundled` because `-DWITH_READLINE=system` was also default option.

Another thing. Above we have `-DWITH_SYSTEM_LIBS=ON` which doesn't have much sense since it probably only sets `-DWITH_CURL=system` and `-DWITH_SSL=system` because the rest (ICU, LIBEVENT, LZ4, PROTOBUF, ZSTD, FIDO) is later changed to `bundled`.